tools/slang-unit-test/unit-test-cooperative-type-metadata.cpp

```
@@ -0,0 +1,163 @@
+    d.Store<CoopMatMatrixLayout::RowMajor>(outputBuffer, 0, 16);
+
+    let dInt = coopMatMulAdd<int32_t, false>(
+        CoopMat<int8_t, MemoryScope.Subgroup, 16, 16, CoopMatMatrixUse::MatrixA>(1),
+        CoopMat<int8_t, MemoryScope.Subgroup, 16, 16, CoopMatMatrixUse::MatrixB>(2),
+        CoopMat<int32_t, MemoryScope.Subgroup, 16, 16, CoopMatMatrixUse::MatrixAccumulator>(3)
+    );
+    dInt.Store<CoopMatMatrixLayout::RowMajor>(outputBufferInt, 0, 16);
+}
+)";
+
+    ComPtr<slang::IGlobalSession> globalSession;
```

Cppcheck (2.19.0)

[style] The function 'wcsnlen_s' is never used.

(unusedFunction)